TCPserver.cpp

```diff
@@ -0,0 +1,63 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <sys/types.h>
+#include <sys/socket.h>
+#define _XOPEN_SOURCE_EXTENDED 1
+#include <unistd.h> //included this for the close function
+
+#include <netinet/in.h>
+
+int main()
+{
+    // data to be sent to cleints
+    char server_message[256] = "Recieved message FUCK";
```

Comment:
This isn't the nicest thing you could say to a client....  But why would you make a buffer of size 256 and only use 20ish chars?